doc/source/user_guide/beta_feature_access.rst

```diff
@@ -0,0 +1,71 @@
+.. _ref_beta_feature_access:
+
+Beta features
+=============
+
+PyFluent provides access to experimental or under-development capabilities of
```

Comment:
I'm not sure if we should call beta features as under-development from Fluent's perspective. In Fluent, beta features are those features for which development is complete, but they are not fully tested or validated.